sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/access/SormasToSormasDiscoveryService.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.backend.sormastosormas.access;
+
+import java.net.MalformedURLException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.symeda.sormas.api.SormasToSormasConfig;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.backend.common.ConfigFacadeEjb.ConfigFacadeEjbLocal;
+import de.symeda.sormas.backend.sormastosormas.SormasToSormasFacadeEjb.SormasToSormasFacadeEjbLocal;
+import io.lettuce.core.ClientOptions;
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.RedisURI;
+import io.lettuce.core.SslOptions;
+import io.lettuce.core.SslVerifyMode;
+import io.lettuce.core.api.sync.RedisCommands;
+
+public class SormasToSormasDiscoveryService {
```

Comment:
Missing annotation to be able to inject it in other Beans.

<details>
<summary>Stack trace</summary>

```
Caused by: javax.naming.NamingException: Lookup failed for 'java:comp/env/de.symeda.sormas.backend.sormastosormas.sharerequest.SormasToSormasShareRequestFacadeEJB/sormasToSormasDiscoveryService' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NamingException: Exception resolving Ejb for 'Remote ejb-ref name=de.symeda.sormas.backend.sormastosormas.sharerequest.SormasToSormasShareRequestFacadeEJB/sormasToSormasDiscoveryService,Remote 3.x interface =de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService,ejb-link=null,lookup=,mappedName=,jndi-name=de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService,refType=Session' .  Actual (possibly internal) Remote JNDI name used for lookup is 'de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService' [Root exception is javax.naming.NamingException: Lookup failed for 'de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NameNotFoundException: de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService not found]]]
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:496)
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:442)
	at java.naming/javax.naming.InitialContext.lookup(InitialContext.java:409)
	at java.naming/javax.naming.InitialContext.lookup(InitialContext.java:409)
	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl._inject(InjectionManagerImpl.java:561)
	... 119 more
Caused by: javax.naming.NamingException: Exception resolving Ejb for 'Remote ejb-ref name=de.symeda.sormas.backend.sormastosormas.sharerequest.SormasToSormasShareRequestFacadeEJB/sormasToSormasDiscoveryService,Remote 3.x interface =de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService,ejb-link=null,lookup=,mappedName=,jndi-name=de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService,refType=Session' .  Actual (possibly internal) Remote JNDI name used for lookup is 'de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService' [Root exception is javax.naming.NamingException: Lookup failed for 'de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NameNotFoundException: de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService not found]]
	at com.sun.ejb.EjbNamingReferenceManagerImpl.resolveEjbReference(EjbNamingReferenceManagerImpl.java:185)
	at com.sun.enterprise.container.common.impl.ComponentEnvManagerImpl$EjbReferenceProxy.create(ComponentEnvManagerImpl.java:1015)
	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:770)
	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:738)
	at com.sun.enterprise.naming.impl.JavaURLContext.lookup(JavaURLContext.java:159)
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:476)
	... 123 more
Caused by: javax.naming.NamingException: Lookup failed for 'de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NameNotFoundException: de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService not found]
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:496)
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:442)
	at java.naming/javax.naming.InitialContext.lookup(InitialContext.java:409)
	at java.naming/javax.naming.InitialContext.lookup(InitialContext.java:409)
	at com.sun.ejb.EjbNamingReferenceManagerImpl.resolveEjbReference(EjbNamingReferenceManagerImpl.java:180)
	... 128 more
Caused by: javax.naming.NameNotFoundException: de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService#de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService not found
	at com.sun.enterprise.naming.impl.TransientContext.doLookup(TransientContext.java:237)
	at com.sun.enterprise.naming.impl.TransientContext.lookup(TransientContext.java:204)
	at com.sun.enterprise.naming.impl.SerialContextProviderImpl.lookup(SerialContextProviderImpl.java:70)
	at com.sun.enterprise.naming.impl.LocalSerialContextProviderImpl.lookup(LocalSerialContextProviderImpl.java:114)
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:483)
	... 132 more
|#]
```
</details>

Reply:
It is a bean now, producer was removed